utils/utils.go

```diff
@@ -277,18 +277,14 @@ func Untar(dst string, r io.Reader, filenames ...string) error {
 
 // Unzip will decompress a zip archive, moving all files and folders
 // within the zip file (parameter 1) to an output directory (parameter 2).
-func Unzip(src string, dest string, fPaths ...string) ([]string, error) {
-
-	var filenames []string
-
+func Unzip(src string, dest string, fPaths ...string) (filenames []string, err error) {
```

Comment:
Please take a look at [PR14](https://github.com/pastelnetwork/pastel-utility/pull/14). it's seem @tsimafeisetchinava000 remove this part

```
if !Contains(fPaths, fpath) && len(fPaths) != 0 {
	continue
}
```

Reply:
PR # 14 is still open. The changes will apply in the scope of that PR